paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -77,6 +103,97 @@ public class PaparazziPlugin : Plugin<Project> {
     }
   }
 
+  /**
+   * Ported from: [ArtifactPublishingUtil.kt](https://cs.android.com/android-studio/platform/tools/base/+/65c212b013f1c94d8db8f488f50a53d6b3ace64f:build-system/gradle-core/src/main/java/com/android/build/gradle/internal/scope/ArtifactPublishingUtil.kt;drc=65c212b013f1c94d8db8f488f50a53d6b3ace64f;l=101)
+   *
+   * Publish intermediate artifacts in the BuildArtifactsHolder in addition to default PublishingSpecs.
+   *
+   * ignore original publishInfo: VariantPublishingInfo? parameter as we are not using publication config
```

Comment:
this can be passed as `null`, since the signature of `com.android.build.gradle.internal.scope.ArtifactPublishingUtil.kt#publishBuildArtifacts` is:

```
fun publishBuildArtifacts(
    creationConfig: ComponentCreationConfig,
    publishInfo: VariantPublishingInfo?
) {
```